<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewOrImportWalletWidget</class>
 <widget class="QWidget" name="NewOrImportWalletWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="newWalletBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>创建钱包</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importWalletBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>导入钱包</string>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>38</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="walletFilePathLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
